Source/SwiftLintFramework/Models/Linter.swift

```diff
@@ -249,6 +249,9 @@ public struct CollectedLinter {
                 "completely removed in a future release.")
         }
 
+        // Free some memory used for this file's caches. They shouldn't be needed after this point.
+        file.invalidateCache()
```

Comment:
I wonder if doing this asynchronously in a fire-and-forget way would speed things up.

```suggestion
        Task {
            file.invalidateCache()
        }
```

Reply:
https://github.com/realm/SwiftLint/pull/4351